app/views/admin/orders/_order_details_table.html.erb

```diff
@@ -0,0 +1,35 @@
+<table class="table table-borderless table-sm">
+  <thead>
+    <tr>
+      <th scope="col">Order Details</th>
+      <th scope="col">Payment Address</th>
+      <th scope="col">Shipping Address</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        Id: <%= order.id %> <br>
+        Created: <%= order.created_at.strftime("%F %R") %><br>
```

Comment:
Może przenieść całe `created_at.strftime("%F %R")` do prezentera ?

Reply:
Ale w teście zostawić tak jak jest, to przy okazji to przetestuje

Reply:
dobry pomysl. Ja lubie robic formatowanie daty przy uzyciu https://guides.rubyonrails.org/v2.3/i18n.html#adding-datetime-formats